woocommerce/woocommerce-android
# Show orders data card in analytics hub

Closes: #5237 (Part of)

### Description
This pull request uses previously created AnalyticsInformationCard to show specific order stats data into new analytics hub

### Testing instructions

1. Open the application compiled in debug mode
2. Click on analytics section on bottom navigation bar
3. Orders card should appear

### Images/gif
<img src="https://user-images.githubusercontent.com/572601/145536296-ecfd7288-0c11-46bd-a225-6496e23a9a87.png" width="389" height="900">
<img src="https://user-images.githubusercontent.com/572601/145536300-ba884816-59a1-49fa-9b49-a48294a9d6bb.png" width="389" height="900">
<img src="https://user-images.githubusercontent.com/572601/145536339-ca7e1094-3a61-45ee-b849-4c9f1bbb033f.png" width="389" height="900">
<img src="https://user-images.githubusercontent.com/572601/145536344-1333f457-503c-4d5e-90dd-8237d6bc1539.png" width="389" height="900">
